<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
                background-color: #E6E6FA;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLineEdit" name="lineEdit_input">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}
</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">QLabel {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}
</string>
           </property>
           <property name="text">
            <string>Enter an expression:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_evaluate">
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color:rgb(255, 184, 162);
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: beige;
        font: bold 14px;
        min-width: 10em;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>Evaluate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget{
        background-color: #E6E6FA;
		border-style: outset;
		border-color: black;
        font: bold 14px;
        padding: 6px;
         gridline-color: black;
         font-size: 12pt;
}

QHeaderView{
     background-color: #E6E6FA;
}

QHeaderView::section {
	background-color: rgb(255, 189, 172);
	padding: 4px;
	font-size: 14pt;
	border-style: none;
	border-bottom: 1px solid black;
	border-right: 1px solid black;
}

QHeaderView::section:horizontal
{
	border-top: 2px solid black;
	border-bottom:	2px solid black;
	border-left: 2px solid black;
	border-right: 2px solid black;
}

QTableWidget::item{
	border-top: 2px solid black;
	border-bottom:	2px solid black;
	border-left: 2px solid black;
	border-right: 2px solid black;
}
QTableWidget::item:selected{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0.9, y2: 0.9,
                                stop: 0 #fff8db, stop: 1 #596fff );
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="midLineWidth">
          <number>3</number>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="4">
          <widget class="QPushButton" name="pushButton_del_all">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>del all</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_x">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_xor">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>^</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButton_z">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_not">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>!</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButton_implication">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>-&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_y">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_konjuct">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>&amp;&amp;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_disjunction">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="pushButton_del">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>del</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_add_left">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>(</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_add_right">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_equal">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>=</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="pushButton_pierce">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string># (pierce arrow)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="pushButton_sheffer">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
                background-color: #FAFAD2;
        border-style: outset;
        border-width: 2px;
        border-radius: 10px;
        border-color: black;
        font: bold 14px;
        padding: 6px;
}</string>
           </property>
           <property name="text">
            <string>| (sheffers's hatch)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
